docs/can-guides/experiment/crud-beginner/2.js

```diff
@@ -2,13 +2,18 @@
 import { todoFixture } from "//unpkg.com/can-demo-models@5";
 todoFixture(3);
 
-import { Component } from "//unpkg.com/can@5/core.mjs";
+import { StacheDefineElement } from "//unpkg.com/can@5/everything.mjs";
 
-Component.extend({
-	tag: "todos-app",
-	view: `
-		<h1>Today’s to-dos</h1>
-	`,
-	ViewModel: {
-	}
-});
+class TodosApp extends StacheDefineElement {
+    static get view() {
+        return `
+            <h1>Today’s to-dos</h1>
+        `;
+    }
+
+    static get define() {
```

Comment:
I included this because I liked being able to say “can-stache-define-element has two main things on it” in the explanation below. I think it makes sense to introduce these concepts together. Otherwise, I would move the `define` explanation down one more section. 🤷‍♂️

Reply:
Yeah, I think that's fine for now.